File: templates/web-python-selenium/pages/landing_page.py
"""Лендинг (главная страница до авторизации)."""
from selenium.webdriver.common.by import By

from pages.base_page import BasePage


class LandingPage(BasePage):
    path = "/"

    # Не xpath по тексту: копирайт меняется чаще вёрстки и ломал бы селектор;
    # текст заголовка проверяется отдельным ассертом в тесте
    TITLE_HEADING = (By.CSS_SELECTOR, "h1")
    LOGIN_BUTTON = (By.ID, "public-login-btn")

File: templates/web-java-selenide/src/test/java/web/pages/LandingPage.java
package web.pages;

import com.codeborne.selenide.SelenideElement;
import io.qameta.allure.Step;

import static com.codeborne.selenide.Selenide.$;
import static com.codeborne.selenide.Selenide.open;

/** Лендинг (главная страница до авторизации). */
public class LandingPage {

    public final SelenideElement titleHeading = $("h1");
    public final SelenideElement loginButton = $("#public-login-btn");

    @Step("Открываем лендинг")
    public LandingPage openPage() {
        open("/");
        return this;
    }
}
